pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,19 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// CephBlockPoolRadosNamespaceSpecMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type CephBlockPoolRadosNamespaceSpecMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemotePool is the name of the secondary cluster pool
+	// +optional
+	RemoteNamespace string `json:"remoteNamespace"`
+	// Mode is the mirroring mode
```

Comment:
```suggestion
	// Mode is the mirroring mode; either pool or image
```

Reply:
Can we add a `validation:Enum` for the expected values?